### PR TITLE
Remove `Drupal8` and `Symfony2` support

### PR DESCRIPTION
**Pull request by jrfnl**
As discussed in #70, as `Drupal8` and `Symfony2` support hasn't been implemented yet anyway, let's remove the classes inferring that there is support for these frameworks to prevent confusion and simplify maintainance.

Note: this can be considered a breaking change as, in the unlikely situation that someone would have set `CmsFramework` to `Drupal8` or `Symfony2` in their custom ruleset, running PHPCS will now throw a fatal error.
In other words, this commit should go into a major release and will need a changelog entry.

### CODE REVIEW

**Comment by jmarcil** [comment]
I'm okay with breaking changes for the 3.0 release.

I also confirm that the is_token_user_input function defined in Utils.php for Drupal 8 was actually an old version of the Utils... 
https://github.com/FloeDesignTechnologies/phpcs-security-audit/blob/7d57b0bd860d4346a5d2f8ae225a0ef0f7f171b8/Security/Sniffs/Drupal7/Utils.php#L47
so not even current. So we might break someone.. but we'll be actually fixing them 😉.
